book/figures/fig-stiff-starter-conversion.tex

```diff
@@ -1,5 +1,5 @@
 \begin{tikzpicture}[node distance = 4cm, auto]
-  \node [start] (init) {Take your regular or liquid starter};
+  \node [start] (init) {Take your regular starter};
```

Comment:
```suggestion
  \node [start] (init) {Take your starter};
```

Reply:
This process could both start with a regular or liquid starter. Both will work. So maybe just `starter`? Should we elaborate this further in another place?